userland/libnrfserialization/serialization_tock.c

```diff
@@ -150,6 +150,8 @@ void ble_serialization_callback (int callback_type, int rx_len, int c, void* oth
 // ser_app_hal_nrf51.c
 //
 
+#define STORM_INT 5
```

Comment:
Oh yeah... `STORM_INT` was being used but not defined anywhere. That's a dummy variable. I have no idea what it should be. Thanks for catching!